### DISCUSSION TITLE
Difference between gamelist.xml and miyoogamelist.xml

### DISCUSSION THREAD

**Discussion by ac3nl** [OnionUI/Onion#531] [Q&A]
On reddit a user mentioned adding "gamelist":"../../Roms/NEOGEO/gamelist.xml",
To the json.

Does this work, and what is the difference between the miyoogamelist.xml method ?

**Comment by Artickw** [comment]
It's the same thing, only the filename is different.

**Reply by ac3nl** [reply]
> It's the same thing, only the filename is different.

do you need to strip the gamelist.xml file. The mame2003plus one is > 4 Mb ?

**Reply by Olywa** [reply]
Does this relate to your other thread? :D
NeoGeo is already set up in the config to look for miyoogamelist so you should not need to touch the config.json.
Just make a copy of your skraper generated 'gamelist' and rename it to 'miyoogamelist'. 4mb is fine but it will take a little longer to process it so stripping the stuff Miyoo does not use is recommended and will also make it easier to find errors.

Did you see the guide here: https://github.com/jimgraygit/Onion/wiki/3.-FAQ-(Frequently-asked-questions)
